samples/revoke_dataset_access.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 Google LLC
```

Comment:
Can we move this to `samples/snippets/revoke_dataset_access.py`? The `samples/` directory is the old way of doing things that we eventually want to migrate away from. (Context: https://github.com/googleapis/python-bigquery/issues/366)

Reply:
done